types/deluge/deluge-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as deluge from "deluge";
```

Comment:
since the package uses the commonjs `export =` syntax, can you please import it using the commonjs syntax `import deluge = require("deluge")`?

Reply:
Thank you for your review! I'm afraid I haven't quite grasped the difference between the different import syntax's. I have pushed the changes you requested.

Reply:
No worries! There are way too many confusing variations these days.